src/NetMQ.Tests/NetMQ.Tests.csproj

```diff
@@ -8,7 +8,7 @@
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">$(PackageTargetFallback);netcoreapp1.0;portable-net45+win8</PackageTargetFallback>
     <IsTestProject>true</IsTestProject>
-    <TargetFrameworks>netcoreapp2.1;net47</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;net47</TargetFrameworks>
```

Comment:
Consider dropping `netcoreapp2.1`.

Reply:
This is how I test netstandard2.0, so to drop netcoreapp2.1 would also mean to drop netstandard2.0